examples/Cpp/ExampleBase/ExampleBase.cpp

```diff
@@ -230,7 +230,13 @@ void ExampleBase::Run()
         }
 
         // Draw current frame
+        #ifdef LLGL_OS_MACOS
+        @autoreleasepool {
```

Comment:
Please apply the braces according to the [Allman coding style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) also for the `autoreleasepool` statement. Rest of the PR looks good.